src/lib.ts

```diff
@@ -344,7 +345,7 @@ function haveCandies(a: Item, b: Item): boolean {
   return Array.from(candiesRequired.values()).every((val) => val === 1);
 }
 
-const rem = mainStat === $stat`Muscle` ? 2 : mainStat === $stat`Mysticality` ? 3 : 4;
+const rem = mainStatMaximizerString === $stat`Muscle` ? 2 : mainStatMaximizerString === $stat`Mysticality` ? 3 : 4;
```

Comment:
This is wrong, as as are a few others.

You should be using `mainStatMaximizerString` to replace `statToMaximizerString(myPrimestat())` (which is a variable of type `String`), and not to replace `mainStat` (which is a variable of type `$stat`)

Reply:
Okay... tried again.